common-lib/src/main/scala/com/gu/mediaservice/lib/ImageIngestOperations.scala

```diff
@@ -88,7 +88,7 @@ case class StorableOptimisedImage(id: String, file: File, mimeType: MimeType, me
 
 
 
-case class BrowserViewableImage(id: String, file: File, mimeType: MimeType, meta: Map[String, String] = Map.empty, mustUpload: Boolean = false) extends ImageWrapper {
+case class BrowserViewableImage(id: String, file: File, mimeType: MimeType, meta: Map[String, String] = Map.empty, isTransformedFromSource: Boolean = false) extends ImageWrapper {
```

Comment:
Could we have a comment to expand on what `isTransformedFromSource` means in the broader context?